src/qp/quant_pdf.py

```diff
@@ -64,6 +72,13 @@ def pad_quantiles(quants, locs):
 
     return quants_out, locs_out
 
+DEFAULT_PDF_CONSTRUCTOR = 'piecewise_linear'
```

Comment:
Selecting this PDF constructor as the default for now, only because there are unit tests with hard coded quant_gen.pdf() output values that will fail if we use a different constructor. 

It would be preferable to use `cdf_spline_derivative`, however, I would like to defer that work to be done by (or in collaboration with) a researcher.